Makefile

```diff
@@ -1,7 +1,11 @@
+SOURCES = rulebook.tex challenges.tex
+
 pdf:
-	latexmk -pdf rulebook.tex
+	$(foreach curr_file,$(SOURCES), latexmk -pdf $(curr_file);)
```

Comment:
Wouldn't it make more sense to split the targets? e.g.
```
all: rulebook.pdf challenges.pdf

%.pdf: %.tex
  latexmk -pdf $<
```

Reply:
i simplified the Makefile a bit based on your suggestion, see f53942b80a73934814e41d51e4b832aee7c1297a and 11206d30f41050c6e26b364f0856a9c5c35c2ddb.